TileStache/Goodies/VecTiles/sort.py

```diff
@@ -32,7 +32,7 @@ def _by_scalerank(feature):
 def _by_population(feature):
     wkb, properties, fid = feature
     value_for_none = -1000
-    population = properties.get('population', value_for_none)
+    population = int(properties.get('population', value_for_none))
```

Comment:
Is it ok if we make it a little safer to handle bad data? I don't trust osm data to always have an integer there. Maybe something like this?

```python
population_str = properties.get('population', value_for_none)
try:
    population = int(population_str)
except ValueError:
    return value_for_none # maybe we call this something like "default_value" instead now?
return population
```